app/Providers/SearchServiceProvider.php

```diff
@@ -179,7 +179,7 @@ public function getIndexForModel($model, $prefix = null)
                 {
                     $prefix = $prefix ?? config('elasticsearch.indexParams.index');
 
-                    return $prefix . '-' . $model::instance()->searchableIndex();
+                    return $prefix . $model::instance()->searchableIndex();
```

Comment:
There are a few other places in the code that add this hyphen to the index names. The assumption is that `ELASTICSEARCH_INDEX` in .env does not include a hyphen at the end of the prefix, and the code accounts for that in a number of places. Removing the hyphen here makes the creation of the indexes out of sycn with the names of the indexes the searches are expecting in other places. Adding this back in makes it work on my local box, and I expect it'll need to be back in here when we move to staging and prod. I'll add this back in.